src/foam/core/auth/test/UserLifecycleTicketTest.js

```diff
@@ -133,10 +136,25 @@ foam.CLASS({
     // 1. LifecycleAwareDAO which will map remove to put lifecycleState DELETED
     // 2. which in turn triggers UserLifecycleDeletedRuleAction to create ticket
     // and delete associated data.
+    Count beforeCount = (Count) ticketDAO.select(COUNT());
     ((DAO) x.get("userDAO")).remove(user);
     user = (User) ((DAO) x.get("userDAO")).find(user.getId());
     test ( user == null || user.getLifecycleState() == LifecycleState.DELETED, "user deleted: " + (user != null ? user.getLifecycleState().toString() : "null"));
 
+    try {
+      Thread.currentThread().sleep(500);
```

Comment:
is half a second enough room for production data set size and server under heavy load?

Reply:
This is only affects testing - as the rule is 'after' and submitted to agency.